kong/plugins/grpc-gateway/schema.lua

```diff
@@ -10,8 +10,39 @@ return {
         {
           proto = {
             type = "string",
+            required = true,
+            default = nil,
+          },
+        },
+        {
+          additional_protos = {
```

Comment:
New fields require discussions.

Reply:
What we need is to cover `google.proto.any` to match https://developers.google.com/protocol-buffers/docs/proto3#json 

It is not possible without knowing descriptors in advance. I understood the conclusion from related discussions
- https://github.com/Kong/kong/issues/7710
- https://github.com/Kong/kong/issues/8725

is: _support for dynamic reflection is expensive_.

This extra field does it statically while avoiding digression of `protojson` from its sister implementations:
- https://pkg.go.dev/google.golang.org/protobuf/encoding/protojson
- https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/json-formatter
- https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.util.json_util

I did not figure any better way to support `any` type. I would be happy to support it in any better "kong-friendly" way.

Reply:
This also requires an entry in the changelog.

Reply:
Notice that adding fields to `schema.lua` means we also need to add a field in `kong/clustering/compat/removed_fields.lua`.
If you're not sure how to do it, please let me know and I'll do it for you.

Reply:
is this right? https://github.com/Kong/kong/pull/8297/commits/36ef019e6ff1cc5de9fe0c5ce0cd809f299d2fff